rasa/core/actions/forms.py

```diff
@@ -316,19 +316,18 @@ def _get_events_since_last_user_uttered(
             (
                 i
                 for i, event in enumerate(reversed(tracker.events))
-                if event == Restarted() or event == tracker.latest_message
+                if event in [Restarted(), tracker.latest_message]
```

Comment:
Function `FormAction._get_events_since_last_user_uttered` refactored with the following changes:

- Replace multiple comparisons of same variable with in operator ([merge-comparisons](https://docs.sourcery.ai/refactorings/merge-comparisons/))
- Inline variable that is immediately returned ([inline-immediately-returned-variable](https://docs.sourcery.ai/refactorings/inline-immediately-returned-variable/))